content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -0,0 +1,80 @@
+---
+reviewers:
+- janetkuo
+title: TTL Controller for Finished Resources
+content_template: templates/concept
+weight: ??
```

Comment:
I'm not sure what weight I should put here.

Reply:
IIUC, weights are used to order pages under the same (sub-)topic.